projects/vehicles/protos/generic/BusSimple.proto

```diff
@@ -27,132 +28,133 @@ PROTO BusSimple [
   hiddenField SFVec3f rearLeftWheelAngularVelocity   0 0 0
 ]
 {
-Robot {
-  translation IS translation
-  rotation IS rotation
-  name IS name
-  controller IS controller
-  controllerArgs IS controllerArgs
-  model "bus"
-  recognitionColors IS recognitionColors # somehow duplicate with the color field, but useful to avoid lua PROTO regeneration when colors change.
-  children [
-    Transform {
-      children IS sensorsSlotCenter
-    }
-    Transform {
-      translation 11.5 0 0.2
-      children IS sensorsSlotFront
-    }
-    Transform {
-      translation 2.5 0 5.3
-      children IS sensorsSlotTop
-    }
-    Transform {
-      translation -6.2 0 1
-      rotation 0 0 1 3.14159
-      children IS sensorsSlotRear
-    }
-    Transform{
-      rotation 0.57735 0.57735 0.57735 2.094395
-      children [
-        BusShape {
-          color IS color
-        } 
-      ]
-    }
-    Transform {
-      translation 6.26 -0.9 0.56
-      rotation 0 1 0 1.5708
-      children [
-        DEF FRONT_SPOT Shape {
-          appearance PBRAppearance {
-            metalness 0
-            roughness 0.3
+  Robot {
+    translation IS translation
+    rotation IS rotation
+    name IS name
+    controller IS controller
+    controllerArgs IS controllerArgs
+    model "bus"
+    recognitionColors IS recognitionColors # somehow duplicate with the color field, but useful to avoid lua PROTO regeneration when colors change.
```

Comment:
```suggestion
    recognitionColors IS recognitionColors # somehow duplicate with the color field, but useful to avoid javascript PROTO regeneration when colors change.
```